src/app/modules/courses/types/course.ts

```diff
@@ -2,6 +2,6 @@ export interface Course {
   id: number;
   title: string;
   creationDate: string;
-  duration: number;
+  duration: string;
```

Comment:
actually duration should be a number, but in future HW (with custom pipe) you will show it as string, so no need to change data model

Reply:
fixed